### PR TITLE
Fix typos - corrected numbered list numbering

### PR DESCRIPTION
**Pull request by rcaron**

### CODE REVIEW

**Comment by sgoings** [comment]
Thanks for the 👀  and follow-up PR. Seems like Markdown + GitHub's method of rendering doesn't correctly identify the bullets between the ordered list as sub bullets, not top-level bullets.

File: README.md [hunk 1]
```diff
@@ -98,10 +98,10 @@ it executes the following algorithm:
 1. Looks for the `ServiceCatalogEntry` 3PR in the catalog
   - If not found, sets the `status` field to `Failed`, adds an appropriate explanation to the `statusDescription` field
     field to a human-readable description of the error, and stops processing
-1. Looks in the `action` field of the claim and takes the appropriate action
+2. Looks in the `action` field of the claim and takes the appropriate action
   - Valid values are `provision`, `bind`, `unbind`, `deprovision`, `create` and `delete`. See [`ServicePlanClaim` documentation](./doc/DATA_STRUCTURES.md#serviceplanclaim) for details on each value
```
Comment (arschles) [inline README.md]: @rcaron can you add 2 more spaces before this and the below `-` chars? I'm not certain, but I believe that should indent them properly

**Comment by arschles** [comment]
looks like the build failed b/c it couldn't lease a cluster for testing purposes

**Comment by rcaron** [comment]
Added the indents.

**Comment by arschles** [comment]
Thanks @rcaron. Merging since this is a docs only fix

**Pull request merged**